docs/ref/models/expressions.txt

```diff
@@ -987,13 +1000,6 @@ calling the appropriate methods on the wrapped expression.
         in :class:`~django.db.models.expressions.Window`. Defaults to
         ``False``.
 
-    .. attribute:: empty_result_set_value
-
-        Tells Django which value should be returned when the expression is used
-        to apply a function over an empty result set. Defaults to
-        :py:data:`NotImplemented` which forces the expression to be computed on
-        the database.
```

Comment:
Please revert moving `empty_result_set_value`.

Reply:
I think I moved this because it looks like most of the attributes are alphabetised. I've reverted that now.